{"name": "Editar um usuário existente", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\nStacktrace:\n\tGetHandleVerifier [0x00007FF735065335+78597]\n\tGetHandleVerifier [0x00007FF735065390+78688]\n\t(No symbol) [0x00007FF734E18FDC]\n\t(No symbol) [0x00007FF734E5F51F]\n\t(No symbol) [0x00007FF734E96FB2]\n\t(No symbol) [0x00007FF734E919CC]\n\t(No symbol) [0x00007FF734E90AA9]\n\t(No symbol) [0x00007FF734DE5AB5]\n\tGetHandleVerifier [0x00007FF73530EEED+2870973]\n\tGetHandleVerifier [0x00007FF735309698+2848360]\n\tGetHandleVerifier [0x00007FF735326973+2967875]\n\tGetHandleVerifier [0x00007FF73508017A+188746]\n\tGetHandleVerifier [0x00007FF73508845F+222255]\n\t(No symbol) [0x00007FF734DE4B28]\n\tGetHandleVerifier [0x00007FF735410668+3925560]\n\tBaseThreadInitThunk [0x00007FF97A067374+20]\n\tRtlUserThreadStart [0x00007FF97AEBCC91+33]\n\n", "trace": "  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"features\\steps\\test_login_steps.py\", line 89, in validar_usuario_administrador\n    context.login_page.validar_usuario_administrador()\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\neo_liberalizados-automacao\\pages\\login_page.py\", line 69, in validar_usuario_administrador\n    elemento = self.driver.find_element(*LoginPageLocators.VALIDAR_ADMINISTRADOR)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 898, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given que o usuário está logado como \"Administrador\"", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\nStacktrace:\n\tGetHandleVerifier [0x00007FF735065335+78597]\n\tGetHandleVerifier [0x00007FF735065390+78688]\n\t(No symbol) [0x00007FF734E18FDC]\n\t(No symbol) [0x00007FF734E5F51F]\n\t(No symbol) [0x00007FF734E96FB2]\n\t(No symbol) [0x00007FF734E919CC]\n\t(No symbol) [0x00007FF734E90AA9]\n\t(No symbol) [0x00007FF734DE5AB5]\n\tGetHandleVerifier [0x00007FF73530EEED+2870973]\n\tGetHandleVerifier [0x00007FF735309698+2848360]\n\tGetHandleVerifier [0x00007FF735326973+2967875]\n\tGetHandleVerifier [0x00007FF73508017A+188746]\n\tGetHandleVerifier [0x00007FF73508845F+222255]\n\t(No symbol) [0x00007FF734DE4B28]\n\tGetHandleVerifier [0x00007FF735410668+3925560]\n\tBaseThreadInitThunk [0x00007FF97A067374+20]\n\tRtlUserThreadStart [0x00007FF97AEBCC91+33]\n\n", "trace": "  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"features\\steps\\test_login_steps.py\", line 89, in validar_usuario_administrador\n    context.login_page.validar_usuario_administrador()\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\neo_liberalizados-automacao\\pages\\login_page.py\", line 69, in validar_usuario_administrador\n    elemento = self.driver.find_element(*LoginPageLocators.VALIDAR_ADMINISTRADOR)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 898, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Validando que o usuário está logado como Administrador", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\nStacktrace:\n\tGetHandleVerifier [0x00007FF735065335+78597]\n\tGetHandleVerifier [0x00007FF735065390+78688]\n\t(No symbol) [0x00007FF734E18FDC]\n\t(No symbol) [0x00007FF734E5F51F]\n\t(No symbol) [0x00007FF734E96FB2]\n\t(No symbol) [0x00007FF734E919CC]\n\t(No symbol) [0x00007FF734E90AA9]\n\t(No symbol) [0x00007FF734DE5AB5]\n\tGetHandleVerifier [0x00007FF73530EEED+2870973]\n\tGetHandleVerifier [0x00007FF735309698+2848360]\n\tGetHandleVerifier [0x00007FF735326973+2967875]\n\tGetHandleVerifier [0x00007FF73508017A+188746]\n\tGetHandleVerifier [0x00007FF73508845F+222255]\n\t(No symbol) [0x00007FF734DE4B28]\n\tGetHandleVerifier [0x00007FF735410668+3925560]\n\tBaseThreadInitThunk [0x00007FF97A067374+20]\n\tRtlUserThreadStart [0x00007FF97AEBCC91+33]\n\n", "trace": "  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"features\\steps\\test_login_steps.py\", line 89, in validar_usuario_administrador\n    context.login_page.validar_usuario_administrador()\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\neo_liberalizados-automacao\\pages\\login_page.py\", line 69, in validar_usuario_administrador\n    elemento = self.driver.find_element(*LoginPageLocators.VALIDAR_ADMINISTRADOR)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 898, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "parameters": [{"name": "context", "value": "<behave.runner.Context object at 0x00000178B8FF2D50>"}], "start": 1744228132674, "stop": 1744228132685}], "start": 1744228132671, "stop": 1744228132695}, {"name": "When navega até a tela de usuários - Perfil", "status": "skipped", "start": 1744228132707, "stop": 1744228132707}, {"name": "When clica no botão \"Editar\"", "status": "skipped", "start": 1744228132708, "stop": 1744228132708}, {"name": "When edita o tipo de perfil do usuário", "status": "skipped", "start": 1744228132708, "stop": 1744228132708}, {"name": "When edita o nome e email do usuário", "status": "skipped", "start": 1744228132708, "stop": 1744228132708}, {"name": "When clica em Salvar para salvar as alterações", "status": "skipped", "start": 1744228132708, "stop": 1744228132708}, {"name": "Then navega até a tela de usuários - Perfil", "status": "skipped", "start": 1744228132708, "stop": 1744228132708}], "start": 1744228132662, "stop": 1744228132708, "uuid": "2b3f27b6-202c-44a6-8c6d-5523cf304524", "historyId": "b7e99a15203a37c717716af50241a5df", "fullName": "Acessar a página de login e testar fluxo de tela de usuários: Editar um usuário existente", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Acessar a página de login e testar fluxo de tela de usuários"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}